From 73bcdb35b84f3c871c42378748e314f1a74f3856 Mon Sep 17 00:00:00 2001
From: tstegema <tstegema@90b65887-3827-0410-9a23-83215b262276>
Date: Wed, 8 Jun 2011 19:52:53 +0000
Subject: [PATCH 0117/1076] LogService.exe; massaiSNMP.dll V1.0.3.16
 eventConfiguration.txt new entries: 1082*
 massai.notification.drivers.BTPDev.* 31*   *,LH_SBD 1092* *,BTPDev*

compDefinition new entries:
APPLICATIONS.LH_SBD.compDefinition
BagTagPrinter.GENERAL.compDefinition
CONVEYOR2_DEVICE.GENERAL.compDefinition



git-svn-id: svn://localhost/SelfServiceCommon/trunk@146 90b65887-3827-0410-9a23-83215b262276
---
 .../APPLICATIONS.LH_SBD.compDefinition.cfg    |  2 +
 .../APPLICATIONS.LH_SBD.compDefinition.csp    |  2 +
 .../BagTagPrinter.GENERAL.compDefinition.cfg  |  8 +++
 .../BagTagPrinter.GENERAL.compDefinition.csp  |  1 +
 ...ONVEYOR2_DEVICE.GENERAL.compDefinition.cfg |  7 +++
 ...ONVEYOR2_DEVICE.GENERAL.compDefinition.csp |  1 +
 .../cpp/LogService/eventConfiguration.txt     | 52 +++++++++++++++++++
 .../cpp/LogService/inc/LogService_def.h       | 23 ++++++--
 .../cpp/LogService/inc/massaiSNMP_def.h       | 24 +++++++--
 9 files changed, 110 insertions(+), 10 deletions(-)
 create mode 100644 SelfServiceCommon/Massai/cpp/LogService/compSource/APPLICATIONS.LH_SBD.compDefinition.cfg
 create mode 100644 SelfServiceCommon/Massai/cpp/LogService/compSource/APPLICATIONS.LH_SBD.compDefinition.csp
 create mode 100644 SelfServiceCommon/Massai/cpp/LogService/compSource/BagTagPrinter.GENERAL.compDefinition.cfg
 create mode 100644 SelfServiceCommon/Massai/cpp/LogService/compSource/BagTagPrinter.GENERAL.compDefinition.csp
 create mode 100644 SelfServiceCommon/Massai/cpp/LogService/compSource/CONVEYOR2_DEVICE.GENERAL.compDefinition.cfg
 create mode 100644 SelfServiceCommon/Massai/cpp/LogService/compSource/CONVEYOR2_DEVICE.GENERAL.compDefinition.csp

diff --git a/SelfServiceCommon/Massai/cpp/LogService/compSource/APPLICATIONS.LH_SBD.compDefinition.cfg b/SelfServiceCommon/Massai/cpp/LogService/compSource/APPLICATIONS.LH_SBD.compDefinition.cfg
new file mode 100644
index 00000000..ec48416c
--- /dev/null
+++ b/SelfServiceCommon/Massai/cpp/LogService/compSource/APPLICATIONS.LH_SBD.compDefinition.cfg
@@ -0,0 +1,2 @@
+AL_LHSBD;AVAILABLE=31001 | 31002 | 31003 | 31004 | 31005;UNAVAILABLE=31000 | 31006 | 31007 | 31008 | 31010 | 31011 | 31012
+APP_LHSBD;AVAILABLE=AL_LHSBD.AVAILABLE & CLA.AVAILABLE & CUSS_CONVEYOR_COMP.OK & CUSS_BTP_COMP
\ No newline at end of file
diff --git a/SelfServiceCommon/Massai/cpp/LogService/compSource/APPLICATIONS.LH_SBD.compDefinition.csp b/SelfServiceCommon/Massai/cpp/LogService/compSource/APPLICATIONS.LH_SBD.compDefinition.csp
new file mode 100644
index 00000000..a659d078
--- /dev/null
+++ b/SelfServiceCommon/Massai/cpp/LogService/compSource/APPLICATIONS.LH_SBD.compDefinition.csp
@@ -0,0 +1,2 @@
+ALL_OK;&;APP_LHSBD.AVAILABLE;APPLICATIONS.LH_SBD.compDefinition
+PARTLY_OK;|;APP_LHSBD.AVAILABLE;APPLICATIONS.LH_SBD.compDefinition
\ No newline at end of file
diff --git a/SelfServiceCommon/Massai/cpp/LogService/compSource/BagTagPrinter.GENERAL.compDefinition.cfg b/SelfServiceCommon/Massai/cpp/LogService/compSource/BagTagPrinter.GENERAL.compDefinition.cfg
new file mode 100644
index 00000000..7f228956
--- /dev/null
+++ b/SelfServiceCommon/Massai/cpp/LogService/compSource/BagTagPrinter.GENERAL.compDefinition.cfg
@@ -0,0 +1,8 @@
+BTP_HW;OK=108200 | 109250;ERROR=108201 | 108202 | 108205 | 108206
+BTP_PAPER;OK=108207 | 108204 | 109200 | 109201 | 109202 | 109203;ERROR=108203 | 109204 | 109205 | 109256
+BTP_PAPER_STATUS;OK=108207;LOW=108204;EMPTY=108203
+BTP_POWER;OK=109250 | 100721;ERROR=100720
+BTP_WRITER;OK=109250;ERROR=109251 | 109252 | 109253 | 109254 | 109255
+BTP_COMP;OK=BTP_WRITER.OK & BTP_PAPER.OK & BTP_HW.OK & BTP_POWER.OK
+CUSS_BTP_COMP(HIDDEN);OK=BTP_COMP.OK
+CUSS_BTP_FEEDER(HIDDEN);OK=BTP_COMP.OK
diff --git a/SelfServiceCommon/Massai/cpp/LogService/compSource/BagTagPrinter.GENERAL.compDefinition.csp b/SelfServiceCommon/Massai/cpp/LogService/compSource/BagTagPrinter.GENERAL.compDefinition.csp
new file mode 100644
index 00000000..372d9801
--- /dev/null
+++ b/SelfServiceCommon/Massai/cpp/LogService/compSource/BagTagPrinter.GENERAL.compDefinition.csp
@@ -0,0 +1 @@
+HARDWARE_OK;&;BTP_COMP.OK;BagTagPrinter.GENERAL.compDefinition
\ No newline at end of file
diff --git a/SelfServiceCommon/Massai/cpp/LogService/compSource/CONVEYOR2_DEVICE.GENERAL.compDefinition.cfg b/SelfServiceCommon/Massai/cpp/LogService/compSource/CONVEYOR2_DEVICE.GENERAL.compDefinition.cfg
new file mode 100644
index 00000000..63a7039d
--- /dev/null
+++ b/SelfServiceCommon/Massai/cpp/LogService/compSource/CONVEYOR2_DEVICE.GENERAL.compDefinition.cfg
@@ -0,0 +1,7 @@
+CON;OK=9000;ERROR=9002 | 9003 | 9004 | 9005 | 9006 | 9011
+CONHW;OK=100015 | 9000;ERROR=100001 | 100002 | 100013 | 100014
+CONHW_SCL;OK=100300 | 9000;ERROR=100301 | 100302 | 100303
+CONHW_SCN;OK=100100 | 9000;ERROR=100101 | 100102 | 100103
+CONHW_SPS;OK=100200 | 9000;ERROR=100201 | 100202 | 100203
+CON_COMP;OK=CON.OK & CONHW.OK & CONHW_SCN.OK & CONHW_SPS.OK & CONHW_SCL.OK
+CUSS_CONVEYOR_COMP(HIDDEN);OK=CON_COMP.OK
\ No newline at end of file
diff --git a/SelfServiceCommon/Massai/cpp/LogService/compSource/CONVEYOR2_DEVICE.GENERAL.compDefinition.csp b/SelfServiceCommon/Massai/cpp/LogService/compSource/CONVEYOR2_DEVICE.GENERAL.compDefinition.csp
new file mode 100644
index 00000000..6a74c251
--- /dev/null
+++ b/SelfServiceCommon/Massai/cpp/LogService/compSource/CONVEYOR2_DEVICE.GENERAL.compDefinition.csp
@@ -0,0 +1 @@
+HARDWARE_OK;&;CON_COMP.OK;CONVEYOR2_DEVICE.GENERAL.compDefinition
\ No newline at end of file
diff --git a/SelfServiceCommon/Massai/cpp/LogService/eventConfiguration.txt b/SelfServiceCommon/Massai/cpp/LogService/eventConfiguration.txt
index 46523c89..0994b0c3 100644
--- a/SelfServiceCommon/Massai/cpp/LogService/eventConfiguration.txt
+++ b/SelfServiceCommon/Massai/cpp/LogService/eventConfiguration.txt
@@ -1088,6 +1088,31 @@
 30012;1280802,TK_CHECKINKIOSK;[STAT][LOG][SNMP:#:1:30];Turkish Airlines Application change state request from unavailable to stopped requested by System Manager
 30013;1050804,TK_CHECKINKIOSK;[STAT][LOG][SNMP:#:1:30];Turkish Airlines Application change state from available to active requestet by TK_CHECKINKIOSK
 
+31000;1190000,LH_SBD;[STAT][LOG][SNMP:#:1:30];LH SelfBagDrop Application initialize
+31001;1040000,LH_SBD;[STAT][LOG][SNMP:#:1:30];LH SelfBagDrop Application init ready
+31002;1050801,LH_SBD;[STAT][LOG][SNMP:#:1:30];LH SelfBagDrop Application activated
+31003;1060000,LH_SBD;[STAT][LOG][SNMP:#:1:30];LH SelfBagDrop Application terminated
+31004;1130000,LH_SBD;[STAT][LOG][SNMP:#:1:30];LH SelfBagDrop Application unavailable
+31005;1120000,LH_SBD;[STAT][LOG][SNMP:#:1:30];LH SelfBagDrop Application available
+31006;1030001,LH_SBD;[STAT][LOG][SNMP:#:1:30];LH SelfBagDrop Application disabled
+31006;1030801,LH_SBD;[STAT][LOG][SNMP:#:1:30];LH SelfBagDrop Application disabled
+31006;1010001,LH_SBD;[STAT][LOG][SNMP:#:1:30];LH SelfBagDrop Application disabled
+31007;1070000,LH_SBD;[STAT][LOG][SNMP:#:1:30];LH SelfBagDrop Application stopped
+31007;1080000,LH_SBD;[STAT][LOG][SNMP:#:1:30];LH SelfBagDrop Application stopped
+31007;1090000,LH_SBD;[STAT][LOG][SNMP:#:1:30];LH SelfBagDrop Application stopped
+31007;1100000,LH_SBD;[STAT][LOG][SNMP:#:1:30];LH SelfBagDrop Application stopped
+31007;1110000,LH_SBD;[STAT][LOG][SNMP:#:1:30];LH SelfBagDrop Application stopped
+31008;1070802,LH_SBD;[STAT][LOG][SNMP:#:1:30];LH SelfBagDrop Application stopped by SM
+31008;1080802,LH_SBD;[STAT][LOG][SNMP:#:1:30];LH SelfBagDrop Application stopped by SM
+31008;1090802,LH_SBD;[STAT][LOG][SNMP:#:1:30];LH SelfBagDrop Application stopped by SM
+31008;1100802,LH_SBD;[STAT][LOG][SNMP:#:1:30];LH SelfBagDrop Application stopped by SM
+31008;1110802,LH_SBD;[STAT][LOG][SNMP:#:1:30];LH SelfBagDrop Application stopped by SM
+31009;1060801,LH_SBD;[STAT][LOG][SNMP:#:1:30];LH SelfBagDrop Application terminated
+31010;1300000,LH_SBD;[STAT][LOG][SNMP:#:1:30];LH SelfBagDrop Application changed state from available to unavailable
+31011;1290000,LH_SBD;[STAT][LOG][SNMP:#:1:30];LH SelfBagDrop Application changed state from initialize to unavailable
+31012;1280802,LH_SBD;[STAT][LOG][SNMP:#:1:30];LH SelfBagDrop Application change state request from unavailable to stopped requested by System Manager
+31013;1050804,LH_SBD;[STAT][LOG][SNMP:#:1:30];LH SelfBagDrop Application change state from available to active requestet by LH_SBD
+
 
 
 9000;10000,CONVEYOR;[STAT][LOG][SNMP:#:1:30];Conveyor available
@@ -1277,6 +1302,16 @@
 108107;massai.notification.drivers.BPPDev.512;[STAT][LOG][SNMP:#:1:30];Boardingpass-printer paper full
 108108;massai.notification.drivers.BPPDev.256;[STAT][LOG][SNMP:#:1:30];Boardingpass-printer error
 
+108200;massai.notification.drivers.BTPDev.1;[STAT][LOG][SNMP:#:1:30];Baggage tag printer available
+108201;massai.notification.drivers.BTPDev.2;[STAT][LOG][SNMP:#:1:30];Baggage tag printer disconnected
+108202;massai.notification.drivers.BTPDev.4;[STAT][LOG][SNMP:#:1:30];Baggage tag printer not initialized
+108203;massai.notification.drivers.BTPDev.16;[STAT][LOG][SNMP:#:1:30];Baggage tag printer out of paper
+108204;massai.notification.drivers.BTPDev.32;[STAT][LOG][SNMP:#:1:30];Baggage tag printer nearly out of paper
+108205;massai.notification.drivers.BTPDev.64;[STAT][LOG][SNMP:#:1:30];Baggage tag printer opened for service (printhead unlocked)
+108206;massai.notification.drivers.BTPDev.128;[STAT][LOG][SNMP:#:1:30];Baggage tag printer busy
+108207;massai.notification.drivers.BTPDev.512;[STAT][LOG][SNMP:#:1:30];Baggage tag printer paper full
+108208;massai.notification.drivers.BTPDev.256;[STAT][LOG][SNMP:#:1:30];Baggage tag printer error
+
 
 109000;10000,GPPDev_FEEDER;[STAT][LOG][SNMP:#:1:30];General-purpose-printer paper stock OK
 109001;10106,GPPDev_FEEDER;[STAT][LOG][SNMP:#:1:30];General-purpose-printer paper stock OK:MEDIA_FULL
@@ -1310,6 +1345,23 @@
 109156;30108,BPPDev_WRITER;[STAT][LOG][SNMP:#:1:30];Boarding pass-printer unavailable: Paper out
 109156;2020108,BPPDev_WRITER;[STAT][LOG][SNMP:#:1:30];Boarding pass-printer unavailable: Paper out
 
+109200;10000,BTPDev_FEEDER;[STAT][LOG][SNMP:#:1:30];Baggage tag printer paper stock OK
+109201;10106,BTPDev_FEEDER;[STAT][LOG][SNMP:#:1:30];Baggage tag printer paper stock OK:MEDIA_FULL
+109202;10105,BTPDev_FEEDER;[STAT][LOG][SNMP:#:1:30];Baggage tag printer paper stock OK:MEDIA_HIGH
+109203;10107,BTPDev_FEEDER;[STAT][LOG][SNMP:#:1:30];Baggage tag printer attention required - paper stock: MEDIA_LOW
+109204;10108,BTPDev_FEEDER;[STAT][LOG][SNMP:#:1:30];Baggage tag printer attention required - paper stock: MEDIA_EMPTY
+109205;30108,BTPDev_FEEDER;[STAT][LOG][SNMP:#:1:30];Baggage tag printer out of paper
+
+109250;10000,BTPDev_WRITER;[STAT][LOG][SNMP:#:1:30];Baggage tag printer available
+109251;30101,BTPDev_WRITER;[STAT][LOG][SNMP:#:1:30];Baggage tag printer unavailable: MEDIA_JAMMED
+109252;30302,BTPDev_WRITER;[STAT][LOG][SNMP:#:1:30];Baggage tag printer unavailable: HARDWARE_ERROR
+109253;30303,BTPDev_WRITER;[STAT][LOG][SNMP:#:1:30];Baggage tag printer unavailable: CRITICAL_SOFTWARE_ERROR
+109254;30304,BTPDev_WRITER;[STAT][LOG][SNMP:#:1:30];Baggage tag printer unavailable: NOT_REACHABLE
+109255;30305,BTPDev_WRITER;[STAT][LOG][SNMP:#:1:30];Baggage tag printer unavailable: NOT_RESPONDING
+109256;30108,BTPDev_WRITER;[STAT][LOG][SNMP:#:1:30];Baggage tag printer unavailable: Paper out
+109256;2020108,BTPDev_WRITER;[STAT][LOG][SNMP:#:1:30];Baggage tag printer unavailable: Paper out
+
+
 200000;massai.notification.platform.start.local;[STAT][LOG][SNMP:#:1:30];Platform started
 200001;massai.notification.platform.start.remote;[STAT][LOG][SNMP:#:1:30];Platform started remote
 200002;massai.notification.platform.shutdown.local;[STAT][LOG][SNMP:#:1:30];Platform shutdown locally
diff --git a/SelfServiceCommon/Massai/cpp/LogService/inc/LogService_def.h b/SelfServiceCommon/Massai/cpp/LogService/inc/LogService_def.h
index e6b4d537..d682b75e 100644
--- a/SelfServiceCommon/Massai/cpp/LogService/inc/LogService_def.h
+++ b/SelfServiceCommon/Massai/cpp/LogService/inc/LogService_def.h
@@ -9,14 +9,27 @@
  *
  ************************************************************************/
 
-/*! \page logservice LogService - Release Notes
+/*! \page LogService.exe - Logging Service
+ *                         writes logging\svc.log logging\platform*.log logging\statistics*.log
+ *                         alternativ solution: massaiSNMP.dll
  *
- *  @version 1.0.3.15
+ *  @version 1.0.3.16
  *
  *  !!! immer auch wenn nötig massaiSNMP_def.h pflegen !!!
  *
  *  \section change1 Change history
  *
+ *  \date 08.06.2011 \li V1.0.3.16
+ *                   \li extension - SteT
+ *                   \li eventConfiguration.txt new entries:
+ *                       1082* massai.notification.drivers.BTPDev.*
+ *                       31*   *,LH_SBD
+ *                       1092* *BTPDev*
+ *                   \li compDefinition new entries:
+ *                       APPLICATIONS.LH_SBD.compDefinition
+ *                       BagTagPrinter.GENERAL.compDefinition
+ *                       CONVEYOR2_DEVICE.GENERAL.compDefinition
+ *
  *  \date 06.04.2011 \li V1.0.3.15
  *                   \li extension - SteT Mantis 18587: TK_CHECKINKIOSK added in eventConfiguration and compSource
  *
@@ -33,7 +46,7 @@
  *                   \li bugfix   - SteT: Mantis0010459: sporadic CORBA COMM_FAILURE exception when sending indiciation to Java-Servant
  *
  *  \date 22.04.2009 \li V1.0.3.10
- *                   \li extension - SteT MANTIS 7952: Temperatur erscheint nicht in Statistikdaten
+ *                   \li extension - SteT MANTIS 7952: entries for temperature added
  *                       Definition added:
  *                       SelfServiceCommon\Massai\cpp\LogService\compSource\SERVICES.LOGGING.compDefinitionTemperature.cfg
  *                       SelfServiceCommon\Massai\cpp\LogService\compSource\SERVICES.LOGGING.compDefinitionTemperature.csp
@@ -107,8 +120,8 @@
 
 #include "Tools/VERSION.H"
 
-#define FILE_VER_BIN     1,0,3,15
-#define FILE_VER_STR     "FileVersion", "1.0.3.15\0"
+#define FILE_VER_BIN     1,0,3,16
+#define FILE_VER_STR     "FileVersion", "1.0.3.16\0"
 
 #define FILEDESCRIPTION  "FileDescription"  ,"Log Service (Ex)\0"
 #define INTERNALNAME     "InternalName"     ,"Log Service (Ex)\0"
diff --git a/SelfServiceCommon/Massai/cpp/LogService/inc/massaiSNMP_def.h b/SelfServiceCommon/Massai/cpp/LogService/inc/massaiSNMP_def.h
index 79988ccf..6178b1ce 100644
--- a/SelfServiceCommon/Massai/cpp/LogService/inc/massaiSNMP_def.h
+++ b/SelfServiceCommon/Massai/cpp/LogService/inc/massaiSNMP_def.h
@@ -9,14 +9,28 @@
  *
  ************************************************************************/
 
-/*! \page massaisnmp massaiSNMP - Release Notes
+/*! \page massaiSNMP.dll - plugin for MS SNMP-Service;
+ *                         writes logging\svc.log logging\platform*.log logging\statistics*.log;
+ *                         sends SNMP-Traps
+ *                         alternativ solution: logservice.exe
  *
  *  !!! immer auch wenn nötig logService_def.h pflegen !!!
  *
- *  @version 1.0.3.15
+ *  @version 1.0.3.16
  *
  *  \section change1 Change history
  *
+ *  \date 08.06.2011 \li V1.0.3.16
+ *                   \li extension - SteT
+ *                   \li eventConfiguration.txt new entries:
+ *                       1082* massai.notification.drivers.BTPDev.*
+ *                       31*   *,LH_SBD
+ *                       1092* *BTPDev*
+ *                   \li compDefinition new entries:
+ *                       APPLICATIONS.LH_SBD.compDefinition
+ *                       BagTagPrinter.GENERAL.compDefinition
+ *                       CONVEYOR2_DEVICE.GENERAL.compDefinition
+ *
  *  \date 06.04.2011 \li V1.0.3.15
  *                   \li extension - SteT Mantis 18587: TK_CHECKINKIOSK added in eventConfiguration and compSource
  *
@@ -33,7 +47,7 @@
  *                   \li bugfix   - SteT: Mantis0010459: sporadic CORBA COMM_FAILURE exception when sending indiciation to Java-Servant
  *
  *  \date 22.04.2009 \li V1.0.3.10
- *                   \li extension - SteT MANTIS 7952: Temperatur erscheint nicht in Statistikdaten
+ *                   \li extension - SteT MANTIS 7952: entries for temperature added
  *                       Definition added:
  *                       SelfServiceCommon\Massai\cpp\LogService\compSource\SERVICES.LOGGING.compDefinitionTemperature.cfg
  *                       SelfServiceCommon\Massai\cpp\LogService\compSource\SERVICES.LOGGING.compDefinitionTemperature.csp
@@ -98,8 +112,8 @@
 
 #include "Tools/VERSION.H"
 
-#define FILE_VER_BIN     1,0,3,15
-#define FILE_VER_STR     "FileVersion", "1.0.3.15\0"
+#define FILE_VER_BIN     1,0,3,16
+#define FILE_VER_STR     "FileVersion", "1.0.3.16\0"
 
 #define FILEDESCRIPTION  "FileDescription"  ,"Massai SNMP (ex)\0"
 #define INTERNALNAME     "InternalName"     ,"Massai SNMP (ex)\0"
-- 
2.41.0.windows.1

